DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -2,25 +2,44 @@
 # SORMAS Development Environment
 
 ## Server
-- Install [your local server](SERVER_SETUP.md)
-- Or install your local development server using [maven cargo](sormas-cargoserver/README.md)
-- Alternatively [SERVER_DEV_SETUP.md](SERVER_DEV_SETUP.md) could be used (at this time not recommended)
+- Use this guide as Dev-Setup **or** refer to [SERVER_SETUP](SERVER_SETUP.md) for a more production-like setup (recommended for core team developers).
```

Comment:
"Use the maven-cargo server as described below **or** refer to [SERVER_SETUP](SERVER_SETUP.md) for a more production-like setup (recommended for core team developers)."